tests/integration/delete/test_delete_command.py

```diff
@@ -45,10 +45,61 @@ def setUpClass(cls):
 
     def setUp(self):
         self.cf_client = boto3.client("cloudformation")
+        self.s3_client = boto3.client("s3")
         self.sns_arn = os.environ.get("AWS_SNS")
         time.sleep(CFN_SLEEP)
         super().setUp()
 
+    @parameterized.expand(
+        [
+            "aws-serverless-function.yaml",
+        ]
+    )
+    @pytest.mark.flaky(reruns=3)
+    def test_s3_options(self, template_file):
+        template_path = self.test_data_path.joinpath(template_file)
+
+        stack_name = self._method_to_stack_name(self.id())
+
+        deploy_command_list = self.get_deploy_command_list(
+            template_file=template_path,
+            stack_name=stack_name,
+            capabilities="CAPABILITY_IAM",
+            image_repository=self.ecr_repo_name,
+            s3_bucket=self.bucket_name,
+            s3_prefix=self.s3_prefix,
+            force_upload=True,
+            notification_arns=self.sns_arn,
+            parameter_overrides="Parameter=Clarity",
+            kms_key_id=self.kms_key,
+            no_execute_changeset=False,
+            tags="integ=true clarity=yes foo_bar=baz",
+            confirm_changeset=False,
+            region="us-east-1",
```

Comment:
Let's don't use hard coded region information. We had this before and this blocks us running these tests in another region. I would recommend reading region information from environment variables or from boto3 session object. If there are other tests in this file which uses hardcoded region information, updating them would be even better!

Reply:
Will also update any other tests that use hardcoded regions.